src/nvim/autocmd.c

```diff
@@ -379,10 +379,7 @@ void do_augroup(char_u *arg, int del_group)
   } else if (STRICMP(arg, "end") == 0) {  // ":aug end": back to group 0
     current_augroup = AUGROUP_DEFAULT;
   } else if (*arg) {  // ":aug xxx": switch to group xxx
-    int i = au_new_group(arg);
-    if (i != AUGROUP_ERROR) {
-      current_augroup = i;
-    }
+    current_augroup = au_new_group(arg);
```

Comment:
`au_new_group()` never fails in Nvim because `ga_grow()` never fails. I think the comment on line 288 should be updated as well?

Reply:
Damn, good catch.